pom.xml

```diff
@@ -46,7 +46,7 @@
         <air.maven.version>3.3.9</air.maven.version>
 
         <dep.antlr.version>4.5.1</dep.antlr.version>
-        <dep.airlift.version>0.128</dep.airlift.version>
+        <dep.airlift.version>0.129-SNAPSHOT</dep.airlift.version>
```

Comment:
@electrum : looks like this will have to updated to 0.130 and airbase to 55 after the recent dependencies changes to airlift?